toolchain/gcc.py

```diff
@@ -52,13 +52,12 @@ def setUp(self):
                              'gmp-devel', 'glibc-devel', 'mpfr-devel',
                              'makeinfo', 'texinfo', 'mpc-devel'])
         else:
-            packages.extend(['glibc-static', 'autogen', 'guile',
-                             'guile-devel', 'libgo', 'libgo-devel',
-                             'libgo-static', 'elfutils-devel',
+            packages.extend(['glibc-static', 'elfutils-devel',
                              'texinfo-tex', 'texinfo', 'elfutils-libelf-devel',
                              'gmp-devel', 'mpfr-devel', 'libmpc-devel',
-                             'gcc-gnat', 'libgnat', 'zlib-devel',
-                             'gettext', 'libgcc', 'libgomp'])
+                             'zlib-devel', 'gettext', 'libgcc', 'libgomp'])
+            if dist.name == 'rhel' and (int(dist.version)==8 and int(dist.release) >= 6):
```

Comment:
@preetithakur you mean these packages are need for  rhel6.0 to rhel8.4 only? and not required for rhel8.5 and above version?
if thats true, it looks ok to me with small correction, as below
`if dist.name == 'rhel' and (int(dist.version) <=8 and int(dist.release) >= 6):`
I feel this looks better

Reply:
Also take care of indentations (length >79).
run pycodestyle and pylint before push and address the issues if found any.